index.js

```diff
@@ -4,22 +4,33 @@
  * @param {Object} store - An object representing a single store. See the instructions for details on its shape.
  * @returns {Object[]} The same `stores` array that was inputted.
  */
-function addNewStore(stores, store) {}
+function addNewStore(stores, store) {
+  let newLenght = stores.push(store);
```

Comment:
We don't have to assign stores.push to a variable, but the store will still be added to the end of stores.